internal/analysis/analysis.go

```diff
@@ -152,14 +154,148 @@ func (a *analysisOrchestrator) CreateWorkspace(ctx context.Context, orgId string
 	return workspaceResponse.ApplicationvndApiJSON201.Data.Id.String(), nil
 }
 
-func (*analysisOrchestrator) RunAnalysis() (*sarif.SarifResponse, error) {
-	var response sarif.SarifResponse
+//go:embed fake.json
+var fakeResponse []byte
+
+func (a *analysisOrchestrator) RunAnalysis(ctx context.Context, orgId string, workspaceId string) (*sarif.SarifResponse, error) {
+	method := "analysis.RunAnalysis"
+	logger := a.logger.With().Str("method", method).Logger()
+	logger.Debug().Msg("API: Creating the scan")
+	org := uuid.MustParse(orgId)
+
+	host := fmt.Sprintf("%s/rest", a.config.SnykApi())
+	a.logger.Debug().Str("host", host).Str("workspaceId", workspaceId).Msg("starting scan")
+
+	client, err := orchestrationClient.NewClientWithResponses(host, orchestrationClient.WithHTTPClient(a.httpClient))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create orchestrationClient: %w", err)
+	}
+
+	flow := scans.Flow{}
+	flow.UnmarshalJSON([]byte(`{"name": "cli_test"}`))
```

Comment:
I forgot to check the error here when I added this line. Could we please add something like:
```suggestion
	err = flow.UnmarshalJSON([]byte(`{"name": "cli_test"}`))
	if err != nil {
		return nil, fmt.Errorf("failed to create scan request: %w", err)
	}
```